samples/libffmpeg/LibffmpegMain.java

```diff
@@ -120,10 +120,10 @@ public static void main(String[] args) {
                 // AVStream* pStream;
                 var pStream = streamsArray.getAtIndex(C_POINTER, i);
                 // AVStream stream;
-                var stream = MemorySegment.ofAddress(pStream, AVStream.sizeof(), session);
+                var stream = MemorySegment.ofAddress(pStream.address(), AVStream.sizeof(), session);
```

Comment:
I understand this is a straight port, but I don't think this is needed any longer. E.g. you can just use `pStream` and pass it to the subsequent getter, I think? (same in other snippets in this sample)